packages/js/email-editor/tsconfig.json

```diff
@@ -0,0 +1,8 @@
+{
+  "extends": "../../../tsconfig.base.json",
```

Comment:
Monorepo js packages are typically have two TS configs - for transpiring into commonJS and ESModule. We need to align with that approach in this package as well. Correspondingly, it's two TS build scripts.

Reply:
Update: This will be done as part of the build configuration update here: https://github.com/orgs/woocommerce/projects/297/views/1?pane=issue&itemId=98392036